docs_source_files/content/remote_webdriver/remote_webdriver_client.de.md

```diff
@@ -87,7 +98,20 @@ caps.version = 67
 driver = Selenium::WebDriver.for :remote, :url => "http://www.example.com", :desired_capabilities => caps
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+const { Builder, Capabilities } = require("selenium-webdriver");
+const firefox = require("selenium-webdriver/firefox")
+var capabilities = Capabilities.firefox();
+(async function helloSelenium() {
+    let driver = new Builder()
+        .setFirefoxOptions(new firefox.Options().headless())
```

Comment:
Why do we need to set `headless`? I think the example could be confusing because if someone runs it and does not see the browser popping up, they may think that something went wrong. 
Can we please remove this? Just to be unform with the other examples.

Reply:
Hi @diemol 
Really apologies for confusing example PR. Hope now solve that issue.